src/pudl/analysis/timeseries_cleaning.py

```diff
@@ -162,15 +164,16 @@ def utc_dataframe_to_aligned(
 @pa.check_types
 def pivot_aligned_timeseries_dataframe(
     aligned_df: DataFrame[AlignedTimeseriesDataFrame],
-    periods: int,
+    periods: int = 24,
```

Comment:
Periods isn't being used in the function anywhere, and the padding seems to have hard-coded 24, so this parameter seems like it should be removed?  Or if other numbers of periods are possible, then the padding should depend on the `periods` parameter.

Reply:
I've removed `periods` from here. There's a couple other places `periods` gets referenced, so I made it a configurable parameter, but it's not something I've ever messed with, so it might not be necessary at all.